project/AppDependencies.scala

```diff
@@ -1,14 +1,22 @@
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
   private val testScope = "test,component"
```

Comment:
The guide says to move to Test

"
AppDependencies

If the dependency has the following “test, it” or just “test” you will need to move to use Test

Eg: 

"uk.gov.hmrc" %% "bootstrap-test-play-30" % boostrapVersion % “test, it”,

Becomes

"uk.gov.hmrc" %% "bootstrap-test-play-30" % boostrapVersion % Test,
"

Reply:
I changed something in build.sbt and it didn't like Test so I changed back.
But I've fixed now, thank you @IanWood-HMRC.